public/lessons/www/css.md

```diff
@@ -54,7 +54,7 @@ What is the CSS doing here?
 * Selectors determine where the styles are applied.
 * Properties determine what those styles are.
 * CSS begins with a selector, followed by curly brackets.
-* Inside the curly brackets is where your styles are declared. 
+* Styles are declared inside the curly brackets. 
```

Comment:
or even better, make it active voice imperative: "Declare styles inside the curly braces"

also FYI brackets are square and braces are curly, never the reverse